src/vm/arm64/stubs.cpp

```diff
@@ -1275,7 +1275,7 @@ void UMEntryThunkCode::Poison()
     m_pTargetCode = (TADDR)UMEntryThunk::ReportViolation;
 
     // ldp x16, x0, [x12]
-    m_code[1] = 0xd42017c0;
+    m_code[1] = 0xa9400190;
```

Comment:
New encoding matches the comment.
Previous encoding was `BRK #0xbe`

Reply:
Thanks for checking!